documentation/static/tutorials/flux-getting-started.md

```diff
@@ -0,0 +1,142 @@
+# Get Started with Flux
+Flux is InfluxData's new data language designed for querying, analyzing, and acting on data stored in InfluxDB.
+Its takes the power of InfluxQL and the TICKscript and combines them into a single, unified data scripting language.
+
+## Setup Flux from scratch
+Setting up the TICK stack with Flux is pretty simple.
+If you're using the InfluxDB Sandbox, it's even simpler.
+For instructions on using Flux with the sandbox, [skip down](#using-flux-with-the-sandbox).
+
+### Install fluxd, Chronograf, and InfluxDB
+Download the nightly builds of fluxd, Chronograf, and InfluxDB to get the most recent versions that include Flux and Flux-dependent functionality.
+Nightly builds for each are available on the [InfluxData Downloads](https://portal.influxdata.com/downloads) page.
+Once downloaded and unpackaged, move the binaries into your system's `$PATH`.
+
+### Configure InfluxDB to use Flux
+Flux is enabled by default in your `influxdb.conf`, the location of which
+[depends on your operating system](https://docs.influxdata.com/influxdb/v1.5/administration/config/#using-the-configuration-file).
+
+In the default config generated by the nightly builds, Flux is configured under
+the `[flux]` section.
+You shouldn't have to change the default settings but can if you prefer.
+
+_**influxdb.conf**_  
+```toml
+# ...
+
+[flux]
+  # Determines whether the RPC service is enabled.
+  enabled = true
+
+  # Determines whether additional logging is enabled.
+  log-enabled = true
+
+  # The bind address used by the Flux RPC service.
+  bind-address = ":8082"
+
+# ...
+```
+
+### Configure the storage service on InfluxDB
+In future versions of InfluxDB, the default `index-version` will be switching from the
+in-memory [Time-Structured Merge Tree (TSM)](https://docs.influxdata.com/influxdb/v1.5/concepts/storage_engine/)
```

Comment:
This is incorrect. The TSM engine stores time series data and is the same across in-mem and TSI indexes. The index change to TSI is okay to suggest here - but it is really orthogonal to `flux`. Regardless, this should read "switch from the in-memory index to TSI"..